src/urban_disco/core.clj

```diff
@@ -105,13 +62,9 @@
     (move-right state)
     state))
 
+(def center (tile/center (base-grid/build-grid)))
 
-(defn set-tile-explored [grid x y]
-  (assoc-in grid [y x :explored] true))
-
-(def center (calc-starting-tile))
-
-(def base-grid (set-tile-explored (build-grid) (:x center) (:y center)))
+(def base-grid (tile/explore (base-grid/build-grid) (:x center) (:y center)))
```

Comment:
I think you want `grid/explore` here. You likely moved these functions to the grid namespace and the old ones were still around in the tile ns so emacs was ok with it.